.css

```diff
@@ -0,0 +1,4 @@
+header{
+    background-color: gray;
+    color: white;
+}
```

Comment:
- [ ] For semantics correct and best practices of Style-sheet , Kindly I recommend you to rename your .`css `file which helps to link Html file Just like this : 
`<link rel="stylesheet" href="Style.css">`

Reply:
Fixed.